src/expr.jl

```diff
@@ -266,13 +266,16 @@ function _internal_node_to_Expr(source, srcrange, head, childranges, childheads,
         # Move parameters blocks to args[2]
         _reorder_parameters!(args, 2)
         if headsym === :dotcall
+            startsym = args[1]
```

Comment:
Naming quibble: this might not be a `Symbol` - it's just the function to be called, so maybe call it `callee` or `funcname` or something.  For example, consider how `(f()).(x,y)` parses.